src/main/java/com/aws/greengrass/deployment/IotJobsHelper.java

```diff
@@ -462,11 +455,11 @@ protected void subscribeToGetNextJobDescription(Consumer<DescribeJobExecutionRes
                     .SubscribeToDescribeJobExecutionAccepted(describeJobExecutionSubscriptionRequest,
                             QualityOfService.AT_LEAST_ONCE, consumerAccept);
             try {
-                subscribed.get(TIMEOUT_FOR_IOT_JOBS_OPERATIONS_SECONDS, TimeUnit.SECONDS);
+                subscribed.get(mqttClient.getMqttOperationTimeout(), TimeUnit.SECONDS);
```

Comment:
Make sure you update the time unit. AFAIK, the operation timeout is milliseconds, not seconds.